packages/rspack/scripts/prepare-container-runtime.js

```diff
@@ -6,11 +6,12 @@ const runtime = fs.readFileSync(
 	path.resolve(__dirname, "../src/runtime/moduleFederationDefaultRuntime.js"),
 	"utf-8"
 );
-const downgradedRuntime = swc.transformSync(runtime, {
+const { code: downgradedRuntime } = swc.transformSync(runtime, {
```

Comment:
@ahabhgk Could you help me to check it?
The output of swc.transformSync is a object like `{code: string, map?: string}`.
Is a mistake in this code before?

Reply:
I don't know, it works before

Reply:
I'm ok with this change as long as it still minimize the code